urls/_2022.py

```diff
@@ -0,0 +1,226 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2022, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# flake8: noqa
+
+MAP_2021 = {
+    # 2022.2 DISTRO
+    'distro/core/qiime2-2022.2-py36-osx-conda.yml':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2022.2/release/qiime2-2022.2-py36-osx-conda.yml',
+    'distro/core/qiime2-2022.2-py36-linux-conda.yml':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2022.2/release/qiime2-2022.2-py36-linux-conda.yml',
+    'distro/core/2022.2':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/distro/core/qiime20212-1614815453.zip',
+    'distro/core/qiime20212-1614815453.zip':
+        'https://qiime2-data.s3-us-west-2.amazonaws.com/distro/core/qiime20212-1614815453.zip',
```

Comment:
```suggestion
    'distro/core/qiime2-2022.2-py38-osx-conda.yml':
        'https://raw.githubusercontent.com/qiime2/environment-files/master/2022.2/release/qiime2-2022.2-py38-osx-conda.yml',
    'distro/core/qiime2-2022.2-py38-linux-conda.yml':
        'https://raw.githubusercontent.com/qiime2/environment-files/master/2022.2/release/qiime2-2022.2-py38-linux-conda.yml',
    'distro/core/2022.2':
        'https://google.com',
    'distro/core/qiime20222-BUILD_ID.zip':
        'https://google.com',
```

The https://google.com URLs are just placeholders, since we don't have 2022.2 vbox images yet (and won't for many more months).